### ISSUE TITLE
accuracy on the test set

### ISSUE THREAD

**Issue by Chanbluky** [zhanglaplace/MobileFaceNet#6]
thanks for you great contribution! i just checked your training log but i saw the final accuracy on your training set is only 64%, can it achieve 99.5% accuracy on LFW dataset?

**Comment by zhanglaplace** [comment]
@Chanbluky  how can you get the final acc on training data.
